
<div style="margin-left: 10px">
<h1>Universidade Federal de Mato Grosso do Sul</h1>
Campus de Campo Grande <br>
Estatística – Prof. Cássio Pinho dos Reis
</div>
</div>


<left><h3>9ª LISTA DE EXERCÍCIOS - parte 1</h3></left> <br>
<b>Turma:</b> Engenharia de Software <br>
<b>RGA:</b> 2021.1906.069-7<br>
<b>Aluno:</b> Maycon Felipe da Silva Mota



---





<h3> Definir as funções que irei usar nos exercicios abaixos </h3>


In [3]:
import math 

def CalcularZCal(proporcaoAmostral, valorP, tamanhoAmostral):
    calculo_1 = proporcaoAmostral - valorP
    calculo_2 = math.sqrt( (valorP*(1-valorP))/tamanhoAmostral)
    resultado = calculo_1/calculo_2
    print(f"O valor ZCal é {resultado:.3f}")
    return resultado



# 1ª Questão – Numa amostra de 500 eleitores selecionados ao acaso de uma cidade, verificou-se que 52% dos eleitores preferem um certo partido. Poderia essa amostra ter sido retirado de uma população que tivesse 50% de eleitores desse partido? Use o nível de 0,05

In [7]:
# h0 = 0.5
# h0 > 0.5 

# RC: 1,96
CalcularZCal(0.50, 0.52, 500)


O valor ZCal é -0.895


Como ZCAL encontrado foi de 0.895, não se pode rejeitar a hipótese de que a proporção de eleitores democratas é 50%, ao nível de 5%.

# 2ª Questão – Lançou-se uma moeda 100 vezes e obteve-se 60 caras. Testar ao nível de 5% se a moeda é honesta

In [14]:
# h0 = 0.5
# h0 diferente de 0.5 

# RC: 1,96
CalcularZCal(0.5, 0.6, 100)


O valor ZCal é -2.041


-2.0412414523193148

In [ ]:
# ZCAL < RC, logo devemos descartar a hipótese da moeda ser honesta. 

# 3ª Questão – Uma pesquisa revelou que das 500 pessoas de um determinado local, 300 preferem uma marca de sabão em pó. Testar ao nível de 4% se pelo menos 70% dessas pessoas prefere esta marca de sabão em pó.

In [17]:
# h0 > 0.7
# h0 < 0.7

# RC: 2,05
CalcularZCal(0.6, 0.7, 500)

# ZCAL > RC, logo podemos concluir que ao menos 70% das pessoas preferem essa marca.


O valor ZCal é -4.880


-4.879500364742665